.github/workflows/fly-validate.yml

```
@@ -8,6 +8,7 @@ jobs:
   validate:
     name: Validate deployment config and build
     runs-on: ubuntu-latest
+    if: false  # Temporarily disabled - remove this line to re-enable
```

actionlint (1.7.10)

[error] constant expression "false" in condition. remove the if: section

(if-cond)